src/content/1.7/development/naming-conventions/_index.md

```diff
@@ -84,6 +102,12 @@ class CustomerController extends AbstractAdminController
 }
 ```
 
+Following our standard action names, for a standard page you should use
```

Comment:
standard is redundant: 
`Following our standard action names, for a default page you should have:` wdty?

Reply:
ping @matks, would you mind to address this comment?

Then I can merge ^^